ruff_sync.py

```diff
@@ -240,6 +260,10 @@ async def sync(
 def main() -> None:
     args = PARSER.parse_args()
     # config = get_config(args.source)
+
+    # Convert non-raw github upstream url to the raw equivalent
+    args.upstream = github_url_to_raw_url(args.upstream)
```

Comment:
You can apply this function by adding this to the `type` parameter to `parser.add_arguement(..., type=github_url_to_raw_url)`

https://docs.python.org/3/library/argparse.html#argparse-type